presets/bootstrap/network.yml

```diff
@@ -34,6 +34,11 @@ databases:
     - repeat: 1
       name: 'db-{{$index}}'
       openPort: '{{add $index 27017}}'
+      compose:
+        deploy:
+            resources:
```

Comment:
@Wayonb could you confirm the best values for the out of the box docker attributes?

I'm wondering if we should keep it empty but add the extension `compose` point. So once we have a bit more flying time with the custom params we can make them official

Reply:
yea, We can leave them empty for now.  At least they can be set if needed.